src/server/local/index.js

```diff
@@ -76,20 +75,17 @@ export default server => {
 
       sendDataFile(res, dataFilePath, next);
     })
-    .get(
-      [homePageDataPath, tipoHomeDataPath],
-      async ({ params }, res, next) => {
-        const { service, variant } = params;
+    .get([homePageDataPath], async ({ params }, res, next) => {
```

Comment:
If there's only 1 path, it no longer needs to be wrapped in an array
```suggestion
    .get(homePageDataPath, async ({ params }, res, next) => {
```

Reply:
Good spot, cheers